src/fhir/1_Questionnaire-C-IDAS.json

```
@@ -101,12 +101,12 @@
       "type": "choice",
       "required": false,
       "linkId": "cs-idas-2",
-      "text": "Sadness - sad expression, sad voice, tearfulness",
+      "text": "<div>In the last week have you observed <b>sadness</b> in your care recipient? (sad expression, sad voice, tearfulness)</div>,
```

@sierramatice same here no HTML in the `text` field, also I believe this one is missing a closing quotation mark.
```suggestion
      "text": "In the last week have you observed sadness in your care recipient? (sad expression, sad voice, tearfulness)",
```